### ISSUE TITLE
retire streamstatstest.wim.usgs.gov

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/StreamStats#742]
With the StreamStats sub-account and infrastructure in place, we should retire this to avoid accidentally using it.

**Comment by aaronstephenson** [comment]
If I remove this DNS record right now, won't @marsmith and @jknewson code break?

**Comment by marsmith** [comment]
Yes , it will.  I did have this thought, can you postpone this until after #741 is complete and we can update our URLs

**Comment by aaronstephenson** [comment]
Sure, no problem. Just let me know when you're ready by adding another comment here addressed to me.

**Comment by jknewson** [comment]
I mentioned this in slack, but figured I should put it here as well. We could probably go ahead and retire the DNS. I've gone already repointed our app urls to point to streamstats/test.streamstats.usgs.gov.
Thanks again for all your help

**Comment by aaronstephenson** [comment]
DNS record for streamstatstest.wim.usgs.gov is now deleted.
